views/homework/components/overview_card_template.hbs

```diff
@@ -55,12 +55,12 @@
     </a>
     <div>
         {{#inArray this.currentUser._id this.archived}}
-        <a href="/homework/asked"><span class="count-badge" data-badge="Archiviert" /></span></a>
+        <span class="count-badge" data-badge="Archiviert" /></span>
         {{else}}
         {{#if ../this.private}}
-        <a href="/homework/archive"><span class="count-badge" data-badge="Entwürfe" /></span></a>
+        <span class="count-badge" data-badge="Entwürfe" /></span>
         {{else}}
-        <a href="/homework/private"><span class="count-badge" data-badge="Gestellt" /></span></a>
+        <span class="count-badge" data-badge="Gestellt" /></span>
```

Comment:
These changes look good but I don't know where the other changes are coming from? Maybe you selected the false target branch for this PR or forked from the wrong base branch?